### ISSUE TITLE
403 Forbidden Error on Debian 10 (Buster) during apt update for https://packages.sury.org/php

### ISSUE THREAD

**Issue by BerndFPV** [oerdnj/deb.sury.org#2161]
Description:

I am encountering a 403 Forbidden error when running apt update on Debian 10 (Buster) for the Sury PHP packages repository. The error message is as follows:

```
Err:4 https://packages.sury.org/php buster InRelease
  403  Forbidden [IP: 2400:52e0:1e00::1081:1 443]
```

Additionally, when I try to access the URL directly via a web browser, I also receive a 403 Forbidden error:

https://packages.sury.org/php/dists/buster/

Steps to Reproduce:

1. Add the Sury PHP repository to the sources list:

```
sudo apt install apt-transport-https lsb-release ca-certificates
echo "deb https://packages.sury.org/php/ $(lsb_release -sc) main" | sudo tee /etc/apt/sources.list.d/sury-php.list
wget -qO - https://packages.sury.org/php/apt.gpg | sudo apt-key add -
```

2. Run apt update:

`sudo apt update`

Expected Behavior:

The apt update command should successfully fetch the package information from the Sury PHP repository without any errors.

Actual Behavior:

The apt update command fails with a 403 Forbidden error as shown above.

System Information:

-     OS: Debian 10 (Buster)
-     Repository URL: https://packages.sury.org/php
-     Error IP: 2400:52e0:1e00::1081:1
-     Browser access to URL: 403 Forbidden error at https://packages.sury.org/php/dists/buster/

Additional Information:

This issue prevents updating and installing packages from the Sury PHP repository on Debian 10 (Buster). It appears to be a permissions or access configuration issue on the repository server.

Thank you for your assistance in resolving this issue.

**Comment by hashar** [comment]
@BerndFPV see #2098 , Debian 10 / Buster has reached its end of life therefore the distribution has been removed from the sury repository.

**Comment by WayneSallee** [comment]
> @BerndFPV see #2098 , Debian 10 / Buster has reached its end of life therefore the distribution has been removed from the sury repository.

But ,,,,,,,,,,,
https://packages.sury.org/php/dists
shows buster as an option.
So that buster link needs to be removed. 


I got the error in my system update, and went to the website, and saw the forbidden, and figured it was probably end of life, but then was perplexed when I saw the buster link 1 directory up. :-)

Wayne Sallee
Wayne@WayneSallee.com
